src/scripts/stores/ApplicationStore.test.js

```diff
@@ -0,0 +1,50 @@
+import { Map } from 'immutable';
+import * as Constants from '../constants/KbcConstants';
+import dispatcher from '../Dispatcher';
+import ApplicationStore from "./ApplicationStore";
+import StorageTokenStoreConstants from '../modules/tokens/constants';
+
+describe('ApplicationStore', function () {
+  it ('Updates token on STORAGE_TOKEN_REFRESH_SUCCESS', function () {
+    const tokenId = '12345';
+    const tokenString = "6596-167303-MUCFxKXSujdGCUSTyYdmgzUNxnwVXSGx0FSSEbGD";
```

Comment:
```suggestion
    const tokenString = "NEW_TOKEN_STRING";
```
vidim, ze ten token uz nie je platny, no radsej by som ho tu neuvadzal